harness/determined/common/experimental/experiment.py

```diff
@@ -289,3 +304,11 @@ def key(ckpt: checkpoint.Checkpoint) -> Any:
 
     def __repr__(self) -> str:
         return "Experiment(id={})".format(self.id)
+
+    @classmethod
+    def _from_bindings(
```

Comment:
Could you please talk me through why this method exists?

Reply:
Ah -- I see your update. In order to support two levels of constructor. Do you prefer this to a single constructor with a "fetch" argument or something? There's something here that feels like a bifurcation in codepaths that intuition says may be hard to maintain.